src/types.ts

```diff
@@ -13,7 +13,16 @@ export type ThirdPartyTag = {
 	loaded?: boolean;
 	onLoad?: () => void;
 	shouldRun: boolean;
+	sourcepointId?: undefined;
```

Comment:
this field should no longer be needed? as all occurrences of sourcepointIds inside `frontend` have been moved into  the `consent-management-platform`

Reply:
Good point!

```suggestion
```